topo_processor/stac/item.py

```diff
@@ -11,12 +11,14 @@ class Item:
     item_output_path: str
     asset_basename: str
     asset_extension: str
+    transformed_asset_extension: str
     content_type: str
     collection: Collection
     stac_item: stac.Item
 
     def __init__(self, path: str, collection: Collection):
         self.path = path
+        self.transformed_data_path = None
```

Comment:
what happens when there is more than one transformation?

Reply:
it depends on where the other transformation happens? Why do we need more than one transformation in one item?

Reply:
There will be lots of transformers one example is cog.

One example could be a tiff metadata adder, that adds or updates tiff metadata.

Could also apply gzip compression to a tiff after it's made

Reply:
i see. then this will overwrites the `transformed_data_path`. should we change it into a list of transformed_data_path?

Reply:
as discussed we will leave it for later because it requires some restructurings